soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -441,11 +443,23 @@ void Randomizer::LoadHintLocations(const char* spoilerFileName) {
     CustomMessageManager::Instance->CreateMessage(
         Randomizer::hintMessageTableID, TEXT_GANONDORF_NOHINT,
         CustomMessage(gSaveContext.ganonText, gSaveContext.ganonText, gSaveContext.ganonText));
+    CustomMessageManager::Instance->CreateMessage(
+        Randomizer::hintMessageTableID, TEXT_SHEIK_1,
+        CustomMessage("{{message}}", "{{message}}", "{{message}}"));
+    CustomMessageManager::Instance->CreateMessage(
+        Randomizer::hintMessageTableID, TEXT_SHEIK_2,
```

Comment:
What does TEXT_SHEIK_1 and TEXT_SHEIK_2 stand for? Could we name these more specifically?

Reply:
TEXT_SHEIK_1 is text ID 0x700F, which triggers when talking to Sheik while you don't have the hookshot, and reads as follows according to CM wiki:

> To save the forest girl, you need another skill...Head to Kakariko Village!

TEXT_SHEIK_2 is text ID 0x7010, which triggers when talking to Sheik after getting hookshot but before clearing Forest Temple, and reads as follows:

> It looks like you have the skills you need...The forest girl is waiting for your help...

Name suggestions are welcome.

Reply:
I think the names you choose yourself are fine.